i18n/fr-FR.yml

```diff
@@ -363,7 +393,7 @@ common:
     home: domicile
     work: lieu de travail
   time:
-    departureArrivalTimes: "{startTime, time, short}—{endTime, time, short}"
```

Comment:
See comment for English file.

Reply:
Addressed in https://github.com/opentripplanner/otp-react-redux/pull/454/commits/20000026bbcfe01539c0e53281a11d4b448b970c